src/Http/Response.php

```diff
@@ -464,10 +456,11 @@ public function __construct(array $options = [])
             $options['charset'] = Configure::read('App.encoding');
         }
         $this->_charset = $options['charset'];
+        $type = 'text/html';
```

Comment:
I am wondering if this default type should be set by `View` instead of here. Might not be worth doing it right now and potentially break some app but worth considering in future.

Reply:
Could be something we do in a future minor release. It shouldn't be a *breaking* change but it would be a small change in behavior.